README.adoc

```diff
@@ -1,48 +1,166 @@
-# Project Starfish 
+= STARFIX : Simplifying The Devflow
+:toc:
+:toc-placement!:
+endif::[]
 
-A  cross platform client side application that would let you open a file in IDE of your choice (vscode, eclipse, intellij, emacs, vi, etc.) and  other cammands locally directly from the browser or file system.This will enable an option to “Open in IDE” through browser extension on various websites like github, gitlab,gitbucket etc similar to one you see in this demo below.
+image:https://github.com/starfixdev/starfix/blob/master/starfix.png[Starfix Logo,title="Starfix Logo"]
 
-image::https://user-images.githubusercontent.com/31308705/79685056-61078280-8253-11ea-8ac1-aab1531ca0ab.gif[Starfish Demo]
+== Intro
+Would it not be nice if..
 
-## Workflow
-- Click on "Open in IDE" on Repository page.
-- Repository is cloned
-- The cloned Repository is opened in the preferred IDE.
+You could open your IDE on any file from any app ? 
+
+OR
+
+Open a repository/file from a browser in your editor of choice ? 
+
+OR
+
+Start a debugger locally from your web browser ? 
+
+**Starfix** is a **cross platform client side application** that would let you **open a file in Editor** of your choice (__vscode, eclipse, intellij, emacs, vi, etc.__) and  other cammands locally **directly from the browser or file system** .This will enable an option to “Open in IDE” through browser extension __on various websites like **github, gitlab,gitbucket** etc__ similar to one you see in this demo below.
+
+== Demo
+
+image:https://github.com/starfixdev/starfix/blob/master/demo.gif[Starfish Demo]
+
+[discrete]
+# INDEX
 
-## Necessary Utilities 
-- A browser extension that injects "Open In IDE" button 
-- Custom URI Handler to handle `ide://` scheme.
-- A native application that clones the repository and opens up the IDE.
-- A configuration file (.yaml) to hold the Preferred IDE and Clone Path.
-- Command line support for application.
+toc::[]
 
-## Targeted Websites:
-- Primary: [github.com](https://github.com/)
-- Secondary: [gitlab.com](https://gitlab.com/)
-- Additional: [bitbucket](https://bitbucket.org/)
-..more can be added
+== Features
+**Starfix** is not a **monolithic** application instead its made up of several __sub-features__ :
 
-## Targeted Platforms:
+=== CLI Tool
+On entering `starfix` on the terminal the Application Launches.This  is basically done by using a `native executable` placed in appropriate directory along with symbolic links.
```

Comment:
not following what you are trying to tell here - what is "the Application" launch about and how is it relevant for the reader/user  ? 

also - what does info about `native executable` and a directory with symoblic links help the user solve ?